src/main/java/appeng/api/networking/crafting/ICraftingCPU.java

```diff
@@ -36,6 +36,11 @@ public interface ICraftingCPU extends IBaseMonitor<IAEItemStack> {
      */
     long getAvailableStorage();
 
+    /**
+     * @return the storage used by an active crafting job in bytes
+     */
+    long getUsedStorage();
```

Comment:
This needs to have a `default` implementation to preserve backwards API compatibility for any old addons

Reply:
Done, thanks for pointing this out